InstanceExport/PowerShell/README.md

```diff
@@ -8,17 +8,17 @@ In order to use it, you should:
 
 2. Reach out to us at help@devresults.com to request an Instance Export Manifest.
 
-3. Save the manifest file in the same directory/folder you have saved the powershell script. It's important to save the file in the JSON format and that you remember what you named the file. In this tutorial the name we used is *manifest.json*.
+3. Save the manifest file in the same directory/folder you have saved the powershell script. It's important to save the file in the JSON format and that you remember what you named the file. In this tutorial the name we used is _manifest.json_.
 
 4. Open a new command line interface (CLI) prompt that supports using of PowerShell commands (depending on your organizational IT policies, you may need to open an elevated/administator prompt, e.g. by right clicking on the Powershell icon and choosing "Run as administrator"). If you don't have PowerShell installed you can follow instructions at [Installing Power Shell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1).
```

Comment:
Do you want to mention the difference between PowerShell 7 and the on-board version?

Reply:
I've added three quotes in the README text to disclaim about the difference of both PowerShells versions. Hope it helps users to always use PS 7